driver-core/src/test/java/com/datastax/driver/core/CaseSensitivityTest.java

```diff
@@ -20,69 +20,38 @@
 import static com.datastax.driver.core.TestUtils.CREATE_KEYSPACE_SIMPLE_FORMAT;
 import static org.testng.Assert.*;
 
-public class CaseSensitivityTest {
-
-    private void assertExists(CCMBridge.CCMCluster c, String fetchName, String realName) {
-        KeyspaceMetadata km = c.cluster.getMetadata().getKeyspace(fetchName);
-        assertNotNull(km);
-        assertEquals(realName, km.getName());
-    }
-
-    private void assertNotExists(CCMBridge.CCMCluster c, String name) {
-        assertNull(c.cluster.getMetadata().getKeyspace(name));
-    }
+@CCMConfig(clusterProvider = "createClusterBuilderNoDebouncing")
+public class CaseSensitivityTest extends CCMTestsSupport {
```

Comment:
Nice, really shows how concise this makes tests that require individual clusters.